webroot/panel/account.php

```diff
@@ -136,13 +136,35 @@
 "<div class='inline'>
 <form action='' method='POST'>
 <input type='hidden' name='form_type' value='loginshell'>
-<input type='text' name='loginshell' placeholder='Login Shell (ie. /bin/bash)' 
-value=" . $USER->getLoginShell() . " required>
+<select id=",">
+  <option>/bin/bash</option>
+  <option>/bin/zsh</option>
+  <option>/bin/csh</option>
+  <option>/bin/tcsh</option>
+  <option>/bin/fish</option>
+  <option>custom</option>
+</select>
+<br>
+
+<input id="," placeholder="," style="," value='loginshell'> 
+<value=" . $USER->getLoginShell() . " required>
```

Comment:
Looks like bad syntax on lines 149-150. `value=` should be part of the `input` tag, not it's own tag.